pkg/csi/cinder/driver_test.go

```diff
@@ -28,7 +28,7 @@ const (
 )
 
 var (
-	vendorVersion = "1.2.2"
+	vendorVersion = "1.3.0"
```

Comment:
I'm curious what's the best practice (for developers) to update `specVersion`, `Version` and `vendorVersion`? Can we document that either in this PR or in a separated one?

Reply:
that's a good suggestion , I will create one for this

Reply:
please note vendorVersion is only used in test...

Reply:
Yeah, I know, but what is it used for? When should a developer bump it?

Reply:
ok, will document in the DOC PR as well

Reply:
@lingxiankong we bump it for each release without fail and also if any major feature/breaking change added in the plugin, then also it needs a bump (but till now we didnt follow diligently for each major feature addition). But ya , we bump the version for each release

Reply:
@ramineni so what's your opinion for this PR? we can talk about future stuffs in that doc PR

Reply:
@jichenjc could you let me know what is the advantage of bumping the spec version ?  looks like we are still not supporting the extra functions that are provided by latest spec.